lib/streamlit/runtime/secrets.py

```diff
@@ -229,6 +230,8 @@ def __getitem__(self, key: str) -> Any:
             raise KeyError(_missing_key_error_message(key))
 
     def __repr__(self):
+        if not runtime.exists():
+            return f"{self.__class__.__name__}(file_path={self._file_path!r})"
```

Comment:
Because it's not obvious from the immediate context, can you add a comment explaining why the runtime needs to exist for the happy path, and also what the meaning of the `if not runtime.exists` path is?